ipatests/test_xmlrpc/test_pwpolicy_plugin.py

```diff
@@ -31,6 +31,49 @@
                                               Declarative)
 
 
+def pwpolicy_cmd(
+        cmd, minlife, cospriority=None,
+        pwpolicy_group=None, pwpolicyadd=False):
```

Comment:
The method arguments are redundant: there is no need for pwpolicyadd=True/False as the `cmd` arg already defines either add or mod. I would remove the `pwpolicyadd` arg and replace `if pwpolicyadd:` with `if cmd == "pwpolicy_add":`